distribution/docs/src/main/jdocs/templates/documentation.ftl

```diff
@@ -67,4 +67,80 @@ include::${project.build.directory}/doc-contents/_contents/config.adoc[]
 [appendix]
 == Application References
 
-<#include "application-reference.ftl">
\ No newline at end of file
+<#include "application-reference.ftl">
+
+// Include iframeResizer to fix scolling issue when displayed in admin-ui.
+++++
+<script type="text/javascript" src="/admin/iframe-resizer/2.6.2/js/iframeResizer.contentWindow.min.js"></script>
+<script type="text/javascript" >
+"use strict";
+
+function makeTocExpandable() {
+	var level = 1;
+
+	while (level < 6) {
+	const tocHeading = 	document.querySelectorAll(".sectlevel" + level + ">li");
+		for (var i = tocHeading.length - 1; i >= 0; i--) {
+			const childLevels = tocHeading[i].querySelector("ul li");
+			if (childLevels) {
+				tocHeading[i].innerHTML ='<a class="expandable">+ </a>' + tocHeading[i].innerHTML + '';
```

Comment:
❓Instead of "massaging" the HTML using `innerHTML`, wonder if it'd be better to create a new `<a>` element and insert it in the DOM. @andrewkfiedler?

Reply:
good question